aiidalab_widgets_base/process.py

```diff
@@ -412,8 +412,12 @@ def update(self):
 
         output_file_path = None
         if 'remote_folder' in self.calculation.outputs:
```

Comment:
I think we are mixing the two approaches of checking a specific condition in this line and then later using a try: except: block to determine what do if some information is missing. Maybe we can just check both conditions in this line or alternatively not check here and just catch both conditions ( no 'output_filename' and no 'remote_folder') via the `KeyError` except.